contributions/catalog/webapps/templates/contribute/contribution_details.html

```diff
@@ -410,10 +415,35 @@ <h4>Last Updated</h4>
 
 {% include 'topnav_javascript.html' %}
 
+<!-- Modal -->
```

Comment:
<img width="501" alt="image" src="https://user-images.githubusercontent.com/4306604/173737816-14489e7e-7a37-4098-ae07-ea0858fedbac.png">

The close icon is overflowing to the next line. Please fix this.

Reply:
updated